### DISCUSSION TITLE
Any plans to port other SBCs?

### DISCUSSION THREAD

**Discussion by inchoat3** [probonopd/MiniDexed#294] [Q&A] [answered]
I haven't been able to get this project out of my head since I came across it a few weeks ago. What you guys are doing here is fantastic and I love you all. Building and playing MiniDexed is the perfect complement to my Vortex Wireless and Microfreak.

I have been wondering as of late, since the Raspberry Pi 4 is currently selling at 2020 toilet paper scalper prices, is there perhaps a plan somewhere down the line to port to one of the other [albeit less-] popular single board computers? A Rock64 can be had for less than $50 at the moment, to say nothing of more exotic options like the Orange Pi and Radxa boards I keep seeing.

I am going to guess this has more to do with Circle being tailored to the pi than anything? Maybe I am barking up the wrong tree.

I know this is in and of itself a large undertaking and I could never do it myself, but I'm curious and the community here seems positive and willing to indulge a little curiosity. Again, you guys are rad.

**Comment by dcoredump** [comment, marked as answer]
Dexed is ported to Teensy ([MicroDexed](https://codeberg.org/dcoredump/MicroDexed/projects) or [MicroDexed-touch](https://codeberg.org/dcoredump/MicroDexed-touch)).

The [engine](https://codeberg.org/dcoredump/Synth_Dexed) itself may work on several platforms.

**Reply by probonopd** [reply]
While it would be nice to see MiniDexed on other SBCs such as the Rock64, the Circle framework we are using it pretty married to the SOCs in the Raspberry Pi. To make things more portable, one option could possibly be to port MiniDexed to a standalone, statically linked executable for Linux, and to use that executable as the `/sbin/init` together with a Linux kernel (but without all the rest of typical Linux distributions - so no systemd, no jack, etc.). But that'd be for someone else to do.

**Reply by inchoat3** [reply]
Thank you for your response. Right now I still don't know enough to do anything like that myself, that doesn't mean I won't loop back to this in like 8 years with some crazy new ideas and somehow find a way.
